src/promptflow-devkit/promptflow/_internal/__init__.py

```diff
@@ -75,9 +75,9 @@
     ResourceType,
 )
 from promptflow._utils.multimedia_utils import (
-    _create_image_from_file,
-    convert_multimedia_data_to_base64,
-    is_multimedia_dict,
+    BasicMultimediaProcessor,
```

Comment:
who will use this imports?

Reply:
Runtime used before, it's no longer necessary, removed.